### ISSUE TITLE
Getting Resource id from ImageView

### ISSUE THREAD

**Issue by Deepanshu2017** [Deepanshu2017/Android_learning#1]
Is there anyway to get the Resource id from the ImageView Object? I just want to get the image id and based on that I want to switch the images in this [file](https://github.com/Deepanshu2017/Android_learning/blob/master/ImageView/app/src/main/java/com/example/deepanshu/imageview/MainActivity.java)

**Comment by codexdelta** [comment]
Have you tried stackoverflow?

**Comment by Deepanshu2017** [comment]
Yes tried to search it on internet but couldn't find any suitable answer.
